addons/mail/__manifest__.py

```diff
@@ -152,7 +152,6 @@
             ('remove', 'mail/static/src/components/activity_menu_container/*'),
             'mail/static/src/js/utils.js',
             ('include', 'mail.assets_messaging'),
-            'mail/static/src/public/*',
```

Comment:
Does removing this line make any difference? As I believe `mail.assets_discuss_public` bundle on which the line is removed is already not used.

Reply:
since I moved the root_widget into the new folder, this line reported `the path "mail/static/src/public/*" did not resolve to anything`